templates/workspaces/base/terraform/azure-monitor/locals.tf

```diff
@@ -0,0 +1,8 @@
+locals {
+  short_workspace_id = substr(var.tre_resource_id, -4, -1)
+  app_insights_name  = "appi-${var.tre_id}-ws-${local.short_workspace_id}"
+  tre_core_tags = {
+    tre_id              = var.tre_id
+    tre_core_service_id = var.tre_id
+  }
```

Comment:
This doesn't look right. Should be a local from the root module named tre_workspace_tags with different set of values.
There're other references for this in other files.